### PR TITLE
Bemt

### PR DESCRIPTION
**Pull request by umbertocrispino**
Work completed. Ready for review.

### CODE REVIEW

File: BEMT.m [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+Vt     = omega*R;
+Vr     = sqrt((V^2)+ (omega*R.*r_ndim).^2);
+phi    = atan((V)./(omega*r)); 
+
+disp('Which theory do you want to use?')
+disp('1 - IMPULSIVE THEORY')
+disp('2 - WHIRLING THEORY WITH SMALL DISTURBANCES')
+disp('3 - WHIRLING THEORY')
+
+l= input(['']);
```
Comment (EttoreSaetta) [inline BEMT.m]: This one should be an input of the function.

File: BEMT.m [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+for j= 1:ns
+    i=0;
+if value(j) < epsi
+   w_t_B(j)= w_t_B(j)-(y(j)/dy(j));
+else
+    
+while [value(j)> epsi , i<max_iter+1]
+    i= i+1;
+    test1(j)= V^2+ (4*w_t_B(j).*((omega*R*r_ndim(j))- w_t_B(j))); %radice della C-15         
+    test2(j)= lam^2+ (4*w_t_B(j)./(Vt)).*(r_ndim(j)-(w_t_B(j)./(Vt))); %radice della C-17
```
Comment (EttoreSaetta) [inline BEMT.m]: All the comments should be in English.

**Review by EttoreSaetta — COMMENTED** [review]
The test case should be in a sub-folder of the `TestCases` folder, so change the name of the folder and then create a new folder inside for your test case.
The documentation folder is `doc` and not `Doc`.

Moreover, you are pointing to the `master` branch. You have to point to the `develop` branch for your PR.

Then change these other things and then you will be approved for the merge.

File: BEMT.m (hunk 2 above)
Comment (umbertocrispino) [reply BEMT.m]: Solved

File: BEMT.m (hunk 1 above)
Comment (umbertocrispino) [reply BEMT.m]: We do the same as 'elican'; if it is necessary we change it.

**Review by EttoreSaetta — APPROVED** [review]

**Pull request merged**